robot/server/server_test.go

```diff
@@ -88,7 +88,6 @@ func TestServer(t *testing.T) {
 		}
 		resp, err := server.GetCloudMetadata(context.Background(), &req)
 		test.That(t, err, test.ShouldBeNil)
-		test.That(t, resp.GetMachinePartId(), test.ShouldEqual, "the-robot-part")
```

Comment:
?

Reply:
drive-by: it' a duplicate line that was added elsewhere. line 94 is the same thing. originally fixed this when the proto repo updated